src/index.html

```diff
@@ -5,8 +5,279 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img class="header__image" src="./images/logo.png"
+        alt="icon">
+      </a>
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__list">
+            <a href="#apple" class="is-active nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#smartphonesAndTelephones" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__list">
+            <a href="#laptopsAndCompurest" class="nav__link"
+            data-qa="hover">
```

Comment:
```suggestion
            <a 
              href="#laptopsAndCompurest" 
              class="nav__link"
              data-qa="nav-hover"
            >
```
![image](https://user-images.githubusercontent.com/102380127/179348829-7cb8b10a-20f2-4aa2-983a-0f2a3be13c36.png)